src/wp-includes/media.php

```diff
@@ -5456,11 +5460,34 @@ function wp_get_loading_attr_default( $context ) {
 		return false;
 	}
 
+	// Conditionally skip lazy-loading on images before the loop.
+	static $special_content_contexts = array(
```

Comment:
Nit-pick: See https://github.com/WordPress/wordpress-develop/pull/4300#discussion_r1194951632, we probably want to align with that and remove the `static`. Not a big deal either way.

Reply:
Following up here, yet another change of course: Let's go back to the original simple approach, see https://github.com/WordPress/wordpress-develop/pull/4300#discussion_r1196900653. We should just use a normal comparison for both values.